### PR TITLE
Serverless changelog — Dec 16, 2024

### PR DESCRIPTION
**Pull request by benironside**
Updates the serverless changelog for this week's release.

Preview: [Changelog](https://docs-content_bk_244.docs-preview.app.elstc.co/guide/en/serverless/current/serverless-changelog.html)

### CODE REVIEW

**Review by KOTungseth — CHANGES_REQUESTED** [review]
There are no breaking changes in Serverless. 

The PRs in this list are either deprecations or enhancements.

File: serverless/pages/serverless-changelog.asciidoc [hunk 1]
```diff
@@ -4,6 +4,59 @@ For serverless API changes, refer to https://www.elastic.co/docs/api/changes[API
 
 For serverless changes in Cloud Console, refer to https://www.elastic.co/guide/en/cloud/current/ec-release-notes.html[Elasticsearch Service Documentation: Release notes].
 
+[discrete]
+[[serverless-changelog-12162024]]
+== December 16, 2024
+
+[discrete]
+[[breaking-changes-12162024]]
```
Comment (KOTungseth) [inline serverless/pages/serverless-changelog.asciidoc:12]:
```suggestion
[[deprecations-12162024]]
```

**Review by KOTungseth — DISMISSED** [review]
One more tiny change, then it's good to go!

**Review by KOTungseth — APPROVED** [review]

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀

**Pull request merged**